templates/cmd/controller/main.go.tpl

```diff
@@ -14,6 +14,7 @@ import (
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	ctrlrt "sigs.k8s.io/controller-runtime"
 	ctrlrtmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+	"github.com/aws/aws-sdk-go/service/{{ .ServiceIDClean }}"
```

Comment:
We usually import as `svcsdk` then we can just do `svcsdk.EndpointsID` later - see https://github.com/aws-controllers-k8s/code-generator/blob/main/templates/pkg/resource/sdk.go.tpl#L14

Reply:
cool, will change to be consistent